_examples/rss_reader.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/jroimartin/gocui"
+
+	"github.com/crazcalm/go-rss-reader/database"
+	"github.com/crazcalm/go-rss-reader"
+	"github.com/crazcalm/go-rss-reader/cli"
+)
+
+func main() {
+	cli.GlobalConfig = cli.NewConfig()
+	config := cli.GlobalConfig
+	err := config.CliParse()
+	if err != nil {
+		log.Panicln(err)
+	}
+	
+
+	if config.DBExist() {
+		_, err = database.Init(fmt.Sprintf("file:%s?_foreign_keys=1", config.GetDBPath()), false)
```

Comment:
I should make a function for constructing the DB string...

Also, use global config